content-security-policy/embedded-enforcement/embedding_csp-header.https.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Embedded Enforcement: Embedding-CSP header.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="../support/testharness-helper.sub.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "name": "Embedding-CSP is not sent if csp attribute is not set on <iframe>.", 
+          "csp": null, "expected": null,"new_csp": null },
+      { "name": "Send Embedding-CSP when csp attribute of <iframe> is not empty.", 
+          "csp": "script-src 'unsafe-inline'", "expected": "script-src 'unsafe-inline'", "new_csp": null },
+      { "name": "Send Embedding-CSP Header on change of `src` attribute on iframe.", 
+          "csp": "script-src 'unsafe-inline'", "expected": "script-src 'unsafe-inline'", "new_csp": "default-src 'unsafe-inline'"},
+      // TODO(amalika): Validate csp attribute to be a proper policy.
+      // { "name": "Send Embedding-CSP Header on change of `src` attribute on iframe.", 
+      //     "csp": "completely wrong csp", "expected": null, "new_csp": null},
```

Comment:
Right now we have no validation of the correctness of the csp attribute so we send Embedding-CSP even if the value is not a proper CSP.